src/backend/tests/app/test_flask_app.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import expungeservice
+from flask import g
+
+
+class TestFlaskApp(unittest.TestCase):
+
+    def setUp(self):
+        self.app = expungeservice.app.create_app('development')
+        self.test_client = self.app.test_client()
+
+    def tearDown(self):
```

Comment:
This isn't necessary if we're not using it. Do you think we can remove it?

Reply:
Sure. I put it in with the assumption that it might get populated later, but that makes more sense to remove it until it's needed.